app/Http/Controllers/Api/V1/SearchController.php

```diff
@@ -144,8 +165,14 @@ public function advance(SearchRequest $searchRequest)
     public function dashboard(SearchRequest $searchRequest)
     {
         $data = $searchRequest->validated();
+
+        $organization = Organization::find($data['organization_id']);
+        $this->authorize('dashboardSearch', $organization);
+
```

Comment:
Can we not use OrganizationRepository instead of eloquent functions directly?

Reply:
Replaced the call to Organization Model functions with repository ones.